tests/tt_metal/tt_metal/unit_tests/basic/runtime_args.cpp

```diff
@@ -39,7 +39,9 @@ uint32_t get_runtime_arg_addr(tt::RISCV processor) {
         } break;
         default: TT_THROW("Unknown processor");
     }
-    return result_base;
+
+    uint32_t offset = is_common ? 3 * 256 * sizeof(uint32_t) : 0;
```

Comment:
these are just in place of a memory allocator and are self contained.  I'll change them to use constexpr so it is more clear what the magic is.  eventually, I'd love for our test infra to use an allocator rather than magic addresses above the base